### PR TITLE
introduce PDF generating and Makefile

### PR DESCRIPTION
**Pull request by pcfe**
addresses redhat-cop/automation-good-practices#21

This MR was created after thinking _oh, I'd like a PDF and I have a Makefile I regularly use anyway_.
Do tell if you would like creation of the HTML tree  added to the `Makefile` and your preferred target name if any.

`{gitdate}` and `{githash}` can be used in the AsciiDoc source

### CODE REVIEW

File: CONTRIBUTE.adoc [hunk 1]
```diff
@@ -98,7 +98,7 @@ Examples:: These are examples
 Even more examples...
 ====
 
-== Publishing
+== Publishing as HTML
```
Comment (ericzolf) [inline CONTRIBUTE.adoc:101]: It's really publishing, meaning to our website at https://redhat-cop.github.io/automation-good-practices/. Please rename more aptly to "Publish for the website".

**Review by ericzolf — CHANGES_REQUESTED** [review]
I'm still not a big friend of makefiles but it can help the one or the other, so I'll aprove the PR once the title is fixed.

File: CONTRIBUTE.adoc (hunk 1 above)
Comment (pcfe) [reply CONTRIBUTE.adoc]: done via 9be882a
feel free to drop 29581f4 I added that one as the link to the website was not immediately obvious to me when lookuing at the source repo.

**Review by ericzolf — APPROVED** [review]

**Comment by jscar-hawk** [comment]
Automation-CoP has approved this PR on 2021-07-28

**Pull request merged**